src/content/windowEventRecorder.js

```diff
@@ -87,7 +87,9 @@ if (!window.loadsterRecorderScriptsLoaded) {
       let element = e.target;
 
       if (recordingOptions.searchForEventListeners && ['click', 'mouseenter', 'mouseover'].includes(e.type)) {
-        element = getElementWithEventListeners(e.target, e.type);
+        if (!e.target.getAttribute('href')) {
```

Comment:
Nice that it's this simple. Should this apply to hover events too though, or only clicks? Seems like when you hover something with an href it's rarely significant.

Reply:
Yeah, makes sense. Thanks!